lib/toTex.js

```diff
@@ -62,6 +64,37 @@ const applyToTex = (node, parent) => {
         throw new Error(`we don't handle 'np' operations yet`);
     } else if (op === 'fact') {
         throw new Error(`we don't handle 'fact' operations yet`);
+    } else if (op === 'nthRoot') {
+        const [radicand, index] = node.args
+          
+        let base, exponent
+        let result
+
+        if (query.isPow(radicand)) {
+            [base, exponent] = radicand.args
+            result = index == '' || query.getValue(index) == 2
+                ? `\\sqrt{${toTex(base, node)}^${toTex(exponent, node)}}`
+                : `\\sqrt[${toTex(index, node)}]{${toTex(base, node)}^${toTex(exponent, node)}}`
+        } else {
+            base = radicand
+            result = index == '' || query.getValue(index) == 2
+                ? `\\sqrt{${toTex(base, node)}}`
+                : `\\sqrt[${toTex(index, node)}]{${toTex(base, node)}}`
+        }
+        return result
+    } else if (op === 'int') {
+        let result
+        if (node.args.length == 2) {
+            const [input, variable] = node.args
+            result = `\\int ${toTex(input, node)} ${toTex(variable, node)}`
```

Comment:
Since the `d` shouldn't be included in `int(x^2, x, 0, 1)` we'll need to add it in here.